core/src/storage/mod.rs

```diff
@@ -47,6 +47,7 @@ mod util {
 }
 
 const BASE_DB_SUBDIR: &str = "base";
```

Comment:
Since there is going to be a resync during migration, maybe rename `base` to `core` to better match the internal naming